script/Ciel/Day 13/13_01_02_CIEL13_7E.txt

```diff
@@ -605,7 +606,7 @@ This True Ancestor has suffered an unimaginable malfunction.
 -- Page 399, Offset 36440.
 -- よりにもよって、一番狂ってはいけない吸血鬼が、一番陥ってはいけない状況に落ちてしまった。
 And as if that wasn't bad enough――
-on top of being the worst possible vampire to have lose control,
+on top of being the worst possible vampire that could have lost control,
  she's now lost it in the worst way possible.
```

Comment:
maybe sneak an 'also' in there
>...she's now also lost...

Reply:
Would prefer this:

> ...she's now also done it in the worst way possible

Reply:
Also + now feels a bit crowded, I'd drop the now if you want to keep the also.

Reply:
Honestly looking at it again I want to remove _more_ particles - 
> she's lost it in the worst possible way.

Reply:
that doesn't work at all for me when I voice it.